cratedb_toolkit/io/cli.py

```
@@ -67,3 +67,60 @@ def load_table(
+    """
+    Export data from CrateDB and CrateDB Cloud clusters.
+    """
+
+    # When `--transformation` is given, but empty, fix it.
+    if transformation is not None and transformation.name == "":
+        transformation = None
+
+    # Encapsulate source and target parameters.
+    source = TableAddress(schema=schema, table=table)
+    target = InputOutputResource(url=url, format=format_, compression=compression)
+    print("target:", target)
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)